### PR TITLE
Fix tag api

### PR DESCRIPTION
**Pull request by RobberPhex**
* Tag can get by id
* on_http_error bring `error_message`, `response_code`, `response_body`

### CODE REVIEW

File: gitlab/v4/objects.py [hunk 1]
```diff
@@ -1035,7 +1035,7 @@ def set_release_description(self, description, **kwargs):
         self.release = server_data
 
 
-class ProjectTagManager(GetFromListMixin, CreateMixin, DeleteMixin,
+class ProjectTagManager(GetMixin, CreateMixin, DeleteMixin,
```
Comment (gpocentek) [inline gitlab/v4/objects.py]: You could use the `NoUpdateMixin` instead of the 4 other mixins.

**Comment by gpocentek** [comment]
Thanks!

The changes look good but you need to re-add the support for listing tags.

**Comment by gpocentek** [comment]
Thanks for the quick update!

**Comment by RobberPhex** [comment]
please release new version. currently, v1.0.0 have this bug.

**Pull request merged**